MagicEightBall/src/components/Result.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import useGetAnswer from "../Utils";
+const Result = ({ question }) => {
+  const answer = useGetAnswer();
```

Comment:
api call should be triggered with respect to form submission, not with component mount. Right now, you have no control if there'll be error from api response.